linux/snap/snapcraft.yaml

```diff
@@ -51,7 +48,8 @@ apps:
       - nextcloud
     command: |
       env LD_LIBRARY_PATH=$SNAP/usr/lib/nextcloud:$LD_LIBRARY_PATH
-      desktop-launch nextcloud
+          FONTCONFIG_FILE=$SNAP/etc/fonts/fonts.conf
+          snapcraft-preload nextcloud
```

Comment:
I did, and it wasn't working in the snapd versions available in trusty and xenial, it should be fixed upstream though...